zotero_sync.py

```
@@ -0,0 +1,274 @@
+- Downloads associated PDFs to the 'to-read' folder
+- Manages sync state and removes tags after processing
+
+"""
+
+import os
+import json
+import logging
+from pathlib import Path
+from typing import Dict, List, Optional
+from pyzotero import zotero
+import requests
```

Ruff (0.12.2)

`requests` imported but unused

Remove unused import: `requests`

(F401)